testfiles/1ZoneUncontrolled3SurfaceZone.idf

```diff
@@ -0,0 +1,397 @@
+!-Generator IDFEditor 1.34
+!-Option OriginalOrderTop UseSpecialFormat
+!-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
+!-      Use '!' comments if they need to be retained when using the IDFEditor.
+!1ZoneUncontrolled3SurfaceZone.idf
```

Comment:
@mjwitte I will go in and delete the file and add it to EnergyPlusDevSupport\DefectFiles\5261 as you requested.  I can also add a new defect to GitHub and assign it to myself if you haven't already done that.  I'll start a new Pivotal Tracker story as well and assign it to myself and show you as the requester.  I'm not sure why it is having convergence problems...